packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -241,7 +241,7 @@ test.serial(
   },
 );
 
-test.serial('writes feed policy to vstorage', async t => {
+test.skip('writes Beta feed policy to vstorage', async t => {
```

Comment:
shouldn't we remove them now that beta is out?

Reply:
ok.

I guess I wanted some feedback on whether to keep testing the old stuff. Thanks.